src/main/java/core/basesyntax/RegistrationServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+
+public class RegistrationServiceImpl implements RegistrationService {
+    private static final int AGE_MIN = 18;
+    private static final int AGE_MAX = 99;
+    private static final int PASS_LENGTH_LIMIT = 6;
+    private static final int LOGIN_LENGTH_LIMIT = 20;
+    private final StorageDao storageDao = new StorageDaoImpl();
+
+    @Override
+    public User register(User user) {
+        if (user.getLogin() == null || user.getPassword() == null) {
+            throw new RuntimeException("Can't be NULL");
+        }
+        if (user.getAge() < AGE_MIN) {
+            throw new RuntimeException("User is younger than 18 years");
+        }
+        if (user.getAge() > AGE_MAX) {
```

Comment:
What about Centenarian?

Reply:
AGE_MAX is 99, it is just random number, it could be 120 or 110...